clusters/howard-moomboo-space/clusters/defau/somtochi.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: gitops.weave.works/v1alpha1
+kind: GitopsCluster
+metadata:
+  annotations:
+    templates.weave.works/create-request: '{"head_branch":"wge-create-branch-whcgf","title":"Creates
+      resource somtochi","description":"This PR creates a cluster-template-development
+      instance","template_name":"cluster-template-development","parameter_values":{"CLUSTER_NAME":"somtochi","K3S_VERSION":"v1.25.3-k3s1","KUBERNETES_VERSION":"1.25.3","NAMESPACE":"defau"},"commit_message":"Creates
+      cluster-template-development instance","credentials":{},"template_namespace":"default","template_kind":"CAPITemplate"}'
+  labels:
+    templates.weave.works/template-name: cluster-template-development
+    templates.weave.works/template-namespace: default
+    weave.works/capi: bootstrap
+  name: somtochi
+  namespace: defau
+spec:
+  capiClusterRef:
+    name: somtochi
+
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  labels:
+    templates.weave.works/template-name: cluster-template-development
+    templates.weave.works/template-namespace: default
+  name: somtochi
+  namespace: defau
+spec:
+  controlPlaneRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+    kind: VCluster
+    name: somtochi
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+    kind: VCluster
+    name: somtochi
+
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+kind: VCluster
+metadata:
+  annotations:
+    kustomize.toolkit.fluxcd.io/prune: disabled
+  labels:
+    cluster.x-k8s.io/cluster-name: somtochi
+    templates.weave.works/template-name: cluster-template-development
+    templates.weave.works/template-namespace: default
+  name: somtochi
+  namespace: defau
+spec:
+  controlPlaneEndpoint:
+    host: somtochi.turkey.local
```

Comment:
I can set you up with an internal loadbalancer to go with this endpoint, or we can point a public domain at the public ip here so you can access the cluster directly from public internet